client/api.go

```diff
@@ -49,6 +51,11 @@ type InitParams struct {
 	ApiVersion  string
 }
 
+type AdditionalAPIKey struct {
+	Key   string
+	Value string
+}
+
```

Comment:
maybe `AdditionalHeader` would be more generally applicable in future code additions than `AdditionalAPIKey`. i'm not sure : / but anything could be a header but not everything is an API key. you're call though.

Reply:
That's a great suggestion, it should be useful in the future. I implemented it and it works as intented :+1: